### ISSUE TITLE
Fody isn't working on UWP librarys

### ISSUE THREAD

**Issue by jlnostr** [Fody/Fody#205]
If you try to use Fody with a library (.winmd or .dll) it will crash:

```
Schweregrad	Code	Beschreibung	Projekt	Datei	Zeile
Fehler		Fody: An unhandled exception occurred:
Exception:
Der Vorgang ist aufgrund des aktuellen Zustands des Objekts ungültig.
StackTrace:
   bei Mono.Cecil.ModuleDefinition.ProcessDebugHeader()
   bei Mono.Cecil.ModuleDefinition.ReadSymbols(ISymbolReader reader)
   bei Mono.Cecil.ModuleReader.ReadSymbols(ModuleDefinition module, ReaderParameters parameters)
   bei Mono.Cecil.ModuleReader.CreateModuleFrom(Image image, ReaderParameters parameters)
   bei Mono.Cecil.ModuleDefinition.ReadModule(Stream stream, ReaderParameters parameters)
   bei Mono.Cecil.ModuleDefinition.ReadModule(String fileName, ReaderParameters parameters)
   bei InnerWeaver.ReadModule() in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\ModuleReader.cs:Zeile 14.
   bei InnerWeaver.Execute() in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\InnerWeaver.cs:Zeile 79.
Source:
Mono.Cecil
TargetSite:
Void ProcessDebugHeader()	FodyTestApp.Data1		
```

This error appears even on a new blank solution, containing the app itself and a seperate class library. You can download a sample solution from OneDrive [here](https://onedrive.live.com/redir?resid=1F1A8D466CC7377F!124860&authkey=!AOc81zYTAcL84Jc&ithint=file%2czip).

**Comment by SimonCropp** [comment]
1. the solution wont even compile for me. 

```
 Severity	Code	Description	Project	File	Line
Error	CS0246	The type or namespace name 'Data2' could not be found (are you missing a using directive or an assembly reference?)	FodyTestApp	C:\Downloads\FodyTestApp\FodyTestApp\FodyTestApp\App.xaml.cs	51
```

And when i fix that i cant replicate your problem

**Comment by jlnostr** [comment]
I didn't update anything. And now it works like expected. Typical developer things ¯\_(ツ)_/¯
Anyway, thanks for the fast response :+1:

**Comment by jlnostr** [comment]
Oh, well. I found out how to reproduce the issue. 

1. Download the solution on OneDrive again (updated it).
2. Compile it. Everything should work as expected.
3. Change the code in the `Fody.RuntimeComponent.Class1` class, e.g increase one number.
4. Compile it. App crashes.

The only fix I found is to recreate it new (see screenshot below):
![screenshot](https://cloud.githubusercontent.com/assets/5391936/10400056/d7b9a3b4-6eb6-11e5-9ddd-e8281cc79f38.png)
Then everything is working as it should.

**Comment by SimonCropp** [comment]
so it fails when u change and build. and then is fixed by a rebuild?

**Comment by jlnostr** [comment]
Correct.

**Comment by jlnostr** [comment]
So, have you found a fix? This odd bug is preventing me from creating an package for the store. :/

**Comment by SimonCropp** [comment]
@jlnostr nope i havnt had time to look at this. how you found a fix?

**Comment by jlnostr** [comment]
I didn't find a fix, neither :/

**Comment by SimonCropp** [comment]
this should be fixed in 1.29.4

**Comment by jlnostr** [comment]
Nice, thanks a lot! :+1:  If Visual Studio would work, I could check it :D
